localStorage.js

```diff
@@ -0,0 +1,62 @@
+/* eslint-disable linebreak-style */
+/* local storage start */
+
+function storageAvailable(type) {
+  let storage;
+  try {
+    storage = window[type];
+    const x = '__storage_test__';
+    storage.setItem(x, x);
+    storage.removeItem(x);
+    return true;
+  } catch (err) {
+    return (
+      err instanceof DOMException
+      && (err.code === 22
+        || err.code === 1014
+        || err.name === 'QuotaExceededError'
+        || err.name === 'NS_ERROR_DOM_QUOTA_REACHED')
+      && storage
+      && storage.length !== 0
+    );
+  }
+}
+
+const hasLS = () => storageAvailable('localStorage');
+
+const nameC = document.querySelector('input[name="name"]');
+const formEmail = document.querySelector('input[name="email"]');
+const message = document.querySelector('textarea[name="message"]');
+
+const formData = {};
+
+function saveToStorage(key, value) {
+  if (hasLS) {
+    formData[key] = value;
+    localStorage.setItem(key, value);
+  }
+}
+
+nameC.onchange = () => {
+  saveToStorage('name', nameC.value);
+};
+formEmail.onchange = () => {
+  saveToStorage('email', formEmail.value);
+};
+message.onchange = () => {
+  saveToStorage('message', message.value);
+};
```

Comment:
- As per the project requirement [ here](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/10_localstorage.md#:~:text=You%20must%20create%20a%20single%20JavaScript%20object%20with%20all%20the%20data%20from%20the%20entire%20form%20and%20save%20it%20in%20local%20storage.%20Do%20not%20create) it would be better to use an obejct that will contain the three input data.

__Kindly make sure to use the same PR for the next review__